src/Tizen.Multimedia.Util/ThumbnailExtractor/ThumbnailExtractor.cs

```diff
@@ -202,14 +202,7 @@ private static Native.ThumbnailExtractCallback GetCallback(TaskCompletionSource<
                 {
                     try
                     {
-                        byte[] tmpBuf = new byte[dataSize];
-                        Marshal.Copy(thumbData, tmpBuf, 0, dataSize);
-
-                        tcs.TrySetResult(new ThumbnailExtractionResult(tmpBuf, thumbWidth, thumbHeight));
-                    }
-                    catch (Exception e)
-                    {
-                        tcs.TrySetException(new InvalidOperationException("[" + error + "] Failed to copy data.", e));
+                        tcs.TrySetResult(new ThumbnailExtractionResult(thumbData, thumbWidth, thumbHeight, dataSize));
```

Comment:
Do you have reason to change?
You need to catch exception and pass with TrySetException
because it is a callback from native

Reply:
OK I'll catch it.